packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -331,7 +331,9 @@ export class SGroup {
   static bracketPos(
     sGroup,
     mol,
-    crossBondsPerAtom: { [key: number]: Array<Bond> }
+    crossBondsPerAtom: { [key: number]: Array<Bond> },
+    remol?,
```

Comment:
What is "remol"? Maybe lets add a type here?

Reply:
Added typing